source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -3119,7 +3122,14 @@
  */
 
 #ifndef FreeRTOS_flush_logging
-    #define FreeRTOS_flush_logging()    if( ipconfigHAS_PRINTF || ipconfigHAS_DEBUG_PRINTF ) do {} while( ipFALSE_BOOL )
+    #define FreeRTOS_flush_logging()                     \
```

Comment:
I wouldn't mind if we remove `FreeRTOS_flush_logging()` totally. It is only called from `FreeRTOS_TCP_WIN.c`, and it was useful while debugging, many years ago.

Reply:
Yeah removing it is also fine from MISRA point of view, also as you mentioned it is only used in TCP_WIN.